### ISSUE TITLE
The parallel download did not success.

### ISSUE THREAD

**Issue by rslu2000** [ashrefm/multi-label-soft-f1#5]
Dear Author,

I followed your demo, then the error happened in the poster image parallel download process.
I found the images data indeed downloaded from IMDB to my laptop, but it seems didn't save into disk（destination path).  
Thus, all cases in the filename were sent in to the error exception. the finally output Information as following:
-------output info------
Downloading...

Download in parallel mode took 5135 seconds.
Success: 0
Errors: 39254
------------------------
all the operation was done in my MacPro notebook. I was sure that the download process was proceeding by watching the activity monitor for the network traffic flow.  However, those poster images just not saved in detination path.

No other process error information, just have the output error results.

I have detail read the download_parallel function in utils.py.  still have no idea how to fix the problem.

Thanks for your help and instruction.

-Raymond Lu 
<img width="958" alt="截圖 2020-07-05 上午11 34 23" src="https://user-images.githubusercontent.com/22634146/86524976-8c455980-beb3-11ea-8b89-ad9cc4ba20ef.png">

**Comment by ashrefm** [comment]
Hello,
Could you please try these two tests to help narrow down where the problem you are encountering may be?
1- Copy and paste any movie poster url in your browser. Do you see an image?
2- Try the download_sequential function instead of download_parallel. Do you see any images being downloaded in the destination folder?

**Comment by rslu2000** [comment]
Hello,
Answer the results of the two tests you mentioned, the result are as following:
1. Test 1 is successful. I can copy the url  and paste to my browser and I can see the image showed on the screen.
2. Test 2 is fail. As I changed the download_parallel as download_sequential function call, I cannot see any images showed in the destination folder.
 
<img width="1360" alt="截圖 2020-07-05 下午8 43 32" src="https://user-images.githubusercontent.com/22634146/86532972-701bd980-bf00-11ea-9028-c69ae298d8ac.png">

**Comment by rslu2000** [comment]
Dear Author,
Finally I fixed the issue that I met. The problem was on certificate issue.  
When I run another python program, the result showed the info like "certificate verify failed: unable to get local issuer certificate". Then I found the solution on stackoverflow. The key step for fixing the certificate problem on Mac OS is  that open the search command box, then type "install Certificates.command" and it will properly install everything i need in system.
![image](https://user-images.githubusercontent.com/22634146/90231095-31f1ce00-de4d-11ea-8b30-af17f2afb464.png)

![image](https://user-images.githubusercontent.com/22634146/90231153-4c2bac00-de4d-11ea-866a-b0bbcc668f61.png)
